src/UhdmAst.cpp

```diff
@@ -854,10 +662,10 @@ AstNode* process_operation(vpiHandle obj_h, UhdmShared& shared) {
         return new AstOr(new FileLine("uhdm"), operands[0], operands[1]);
     }
     case vpiBitXorOp: {
-        return new AstXor(new FileLine("uhdm"), operands[1], operands[0]);
+        return new AstXor(new FileLine("uhdm"), operands[0], operands[1]);
     }
     case vpiBitXnorOp: {
-        return new AstXnor(new FileLine("uhdm"), operands[1], operands[0]);
+        return new AstXnor(new FileLine("uhdm"), operands[0], operands[1]);
```

Comment:
Could you add a test case for those, preferably with asserts, to make sure we keep those indices right?